client/src/components/TextArea/Text.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import './Text.css'
+
+
+const Text = (props) => <div>
+<p className={props.pClassList}>Is there anything else you want us to know?</p>
+<textarea className={props.textAreaClassList}>
+Tell us about your dream dress! Do you have any Modern Trousseau favorites?
```

Comment:
Instead of using static text here please pass in a prop here that dictates the text that should be rendered

Reply:
+1

the label text, value, etc should all be passed in as props